index.d.ts

```diff
@@ -271,6 +273,46 @@ declare module '@primer/components' {
 
   export const Link: React.FunctionComponent<LinkProps>
 
+  export interface AnchoredPositionHookSettings extends Partial<PositionSettings> {
```

Comment:
This is just temporary until we remove the index.d.ts file, but I thought it might be nice to export the component here for now so we can test it out in Memex